latex2sympy.py

```diff
@@ -717,3 +709,34 @@ def get_differential_var_str(text):
     if text[0] == "\\":
         text = text[1:]
     return text
+
+
+class Div(sympy.Mul):
+
+    def __new__(cls, *args, in_parsing=False, **options):
+        if in_parsing:
+            args = (args[0], sympy.Pow(args[1], -1, evaluate=False))
+        return super().__new__(Div, *args, **options)
+
+    def _sympyrepr(self, expr, order=None):
+        return "Div(%s)" % ",".join(map(sympy.srepr, self.args))
+
+
+class Root(sympy.Pow):
```

Comment:
I feel like this class should be doing the reverse power itself, it doesn't really make sense to have it behave the same as `Pow` when it's supposed to be the opposite.

Reply:
@tragessere i haven't tested it, but im pretty sure this was to have a non-evaluated exp for sqrt and root, while also giving a custom output for `srepr`